Add meta description to some pages (#4534)
wazuh/wazuh-documentation: source/deploying-with-ansible/roles/wazuh-filebeat.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: Filebeat is used in conjunction with Wazuh Manager to send events and alerts to Elasticsearch. Learn how to customize the installation here.

Reviewer: ```suggestion
  :description: Filebeat is used in conjunction with the Wazuh manager to send events and alerts to Elasticsearch. Learn how to customize the installation here.
```

>   :description: Filebeat can be used in conjunction with Wazuh Manager to send events and alerts to Elasticsearch. Learn how to customize the installation here. 

Reviewer 2: ```suggestion
  :description: Filebeat is used in conjunction with Wazuh Manager to send events and alerts to Elasticsearch. Learn how to customize the installation here.
```
